Update readme with labextension info (#187)
jupyterhub/jupyter-server-proxy: README.rst

   :target: https://www.npmjs.com/package/@jupyterlab/server-proxy

Jupyter Server Proxy lets you run arbitrary external processes (such
as RStudio, Shiny Server, Syncthing, PostgreSQL, Code Server, etc)
alongside your notebook server and provide authenticated web access to
them using a path like `/rstudio` next to others like `/lab`.

> Alongisde the python package that provide the main functionality, the

Reviewer: ```suggestion
Alongside the python package that provides the main functionality, the
```

> JupyterLab extension (`@jupyterlab/server-proxy`) provide buttons in

Reviewer: ```suggestion
JupyterLab extension (`@jupyterlab/server-proxy`) provides buttons in
```
This is an RST file so I don't think the single backticks do what you intended (format as code?). You could try double backticks but run a sphinx/rst build to verify.

`The documentation <https://jupyter-server-proxy.readthedocs.io/>`_
contains information on installation & usage.

Install
=======

Python package
--------------

pip
^^^

.. code-block::

   pip install jupyter-server-proxy

conda
^^^^^

.. code-block::

   conda install jupyter-server-proxy -c conda-forge

JupyterLab extension
--------------------

> 

Reviewer: Since you're updating this, do you think it's worth mentioning you also need to pip-install the python package too, or is this already clear enough?

Author: ```suggestion

Note that as the JupyterLab extension only is a graphical interface to launch registered applications in the python package, the extension requires the python package to be installed.

```

`The documentation <https://jupyter-server-proxy.readthedocs.io/>`_
contains information on installation & usage.

Install
=======

Python package
--------------

pip
^^^

.. code-block::

   pip install jupyter-server-proxy

conda
^^^^^

.. code-block::

   conda install jupyter-server-proxy -c conda-forge

JupyterLab extension
--------------------

Note that as the JupyterLab extension only is a graphical interface to
launch registered applications in the python package, the extension
requires the python package to be installed.

.. code-block::

   jupyter labextension install @jupyterlab/server-proxy

Contributing
============

Python package
--------------

.. code-block::

   pip install -e .

>    # explicit install needed with editable mode (``-e``) jupyter

Reviewer: ```suggestion
   # explicit install needed with editable mode (-e) jupyter
```
This is already inside a code-block:
![image](https://user-images.githubusercontent.com/1644105/78180964-76ab3880-745b-11ea-96c8-a79f04f4ab0c.png)
